keps/sig-node/2033-kubelet-in-userns-aka-rootless/kep.yaml

```diff
@@ -42,3 +43,6 @@ feature-gates:
     components:
       - kubelet
 disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics: []
```

Comment:
As described in the comment this is required field.

Reply:
Answered as N/A